src/Sulu/Bundle/ContentBundle/Admin/ContentContentNavigationProvider.php

```diff
@@ -82,7 +78,7 @@ public function getNavigationItems(array $options = array())
             $permissions->setComponentOptions(
                 array(
                     'display' => 'form',
-                    'type' => Structure::class,
+                    'type' => WebspaceBehavior::class,
```

Comment:
@dantleech I decided to use the `WebspaceBehavior`, since these links are webspaces. IMHO using the `PageBehavior` makes even less sense if I think about your modular UI approach, so people might not want to implement this interface. After thinking about it this behavior is the only one which is really necessary.